packages/office-ui-fabric-react/src/components/Button/CommandButton/CommandButton.tsx

```diff
@@ -27,9 +28,10 @@ export class CommandButton extends BaseComponent<IButtonProps, {}> {
   public render() {
     return (
       <BaseButton
-        classNames={ CLASS_NAMES }
+        { ...this.props }
+        classNames={ assign(CLASS_NAMES, this.props.classNames) }
```

Comment:
you're using assign wrong. First param is the target. In this case you're mixing stuff into the constant definition.